backend/src/main/java/com/htecgroup/skynest/util/UrlUtil.java

```diff
@@ -39,4 +45,23 @@ public class UrlUtil {
               new String[] {USERS_CONTROLLER_URL + LOG_IN_URL})
           .flatMap(Stream::of)
           .toArray(String[]::new);
+
+  public static String getEmailVerificationLink(String token) {
+    return String.format(
+        "%s%s%s%s%s",
+        getBaseUrlBackend(), USERS_CONTROLLER_URL, CONFIRM_EMAIL_URL, "?token=", token);
+  }
+
+  public static String getPasswordResetLink(String token) {
+    return String.format("%s%s%s%s", BASE_URL_FRONTEND, "/confirm-password", "?token=", token);
```

Comment:
Since you already have constants for other urls, do the same here.. and maybe define them as path, cause it is just the part of the url , right?

Reply:
Yeah, everything in the class is just the part of the url, will look to rename everything.